core/native/src/-NonJvmPlatform.kt

```diff
@@ -30,7 +30,9 @@ public actual open class IOException actual constructor(
 ) : Exception(message, cause) {
     public actual constructor(message: String?) : this(message, null)
 
-    public actual constructor() : this(null)
+    public actual constructor(cause: Throwable?) : this(null, cause)
```

Comment:
The usual behavior of this exception constructor is `this(cause?.toString(), cause)`.

Reply:
I'd say it should be super(cause), and all other ctors should delegate to super ctors too.

Like here https://github.com/JetBrains/kotlin/blob/master/libraries/stdlib/wasm/src/kotlin/Exceptions.kt

Reply:
I'll send a new PR after this gets merged.

Reply:
It's in #281.